common_examples/simple-ethernet/main/idf_component.yml

```diff
@@ -4,4 +4,8 @@ dependencies:
   ethernet_init:
     version: '*'
     # For local development use the local copy of the component
-    override_path: '../../../ethernet_init/'
\ No newline at end of file
+    override_path: '../../../ethernet_init'
+  espressif/lan867x:
```

Comment:
Why need to add LAN867x to examples? Shouldn't it be pulled by `Ethernet Init` component?

Reply:
You are right, no need to have it. Removed.